tsfc/driver.py

```diff
@@ -257,13 +257,15 @@ def name_multiindex(multiindex, name):
     return builder.construct_kernel(kernel_name, impero_c, parameters["precision"], index_names, quad_rule)
 
 
-def compile_expression_at_points(expression, points, coordinates, interface=None, parameters=None, coffee=True):
+def compile_expression_at_points(expression, points, to_element, coordinates, interface=None,
+                                 parameters=None, coffee=True):
     """Compiles a UFL expression to be evaluated at compile-time known
     reference points.  Useful for interpolating UFL expressions onto
     function spaces with only point evaluation nodes.
 
     :arg expression: UFL expression
     :arg points: reference coordinates of the evaluation points
+    :arg to_element: UFL element to interpolate onto
```

Comment:
This new argument seems to be ignored.

Reply:
I agree with @miklos1, I don't think this is needed at all. Permission to update and remove so we can move forward with https://github.com/firedrakeproject/firedrake/pull/1453?